### ISSUE TITLE
edges should be included in paths

### ISSUE THREAD

**Issue by ekg** [ekg/vg#56]
The protobuf format doesn't currently have a way to represent when edges are part of paths.

Case in point (from the test directory):

```shell
➜  test git:(master) ✗ vg construct -r tiny/tiny.fa >t.vg; vg align -s CAAATAAGGCTTGGAAATTATATTCCAACTCTCTT -Q query t.vg | vg mod -i - t.vg | vg view -
H       HVN:Z:1.0
S       2       CAAATAAGGCTTGGAAATT
P       2       x       +       19M
L       2       -       5       +       0M
L       2       -       4       +       0M
S       4       TTCTGGAGTTCTATT
P       4       x       +       15M
L       4       -       5       +       0M
S       5       ATATTCCAACTCTCTG
P       5       x       +       16M
```

Nothing in the GFA output (or other output) can refer to the added path `query`. This is a problem with the schema itself.

**Comment by ekg** [comment]
The other way to resolve this is to store the path as a literal list of nodes that are traversed (or mappings against nodes, in `vg` terminology).

**Comment by adamnovak** [comment]
@ekg Do we still want to do this?

**Comment by jeizenga** [comment]
Aren't paths already a list of nodes that are traversed?

**Comment by adamnovak** [comment]
Yes, they are. We have mappings, and when two adjacent mappings are on either side of an edge, the edge is traversed.

I'm not really sure what problem Erik's original example is getting at. I thought this issue was about adding support for a path to be e.g. just an edge, but now I'm not so sure.

**Comment by faithokamoto** [comment]
Doesn't seem like this needs to be done, or at least if there's an undone part it wasn't clarified at the time.
